main.py

```diff
@@ -40,156 +38,130 @@
 from helpers.vertex_ai_service import LLMParameters
 from helpers.bq_service import BigQueryService
 from configuration import Configuration
+from utils.utils import parse_args, build_abcd_params_config
```

Comment:
`utils.utils` sounds a bit repetitive. Can this be moved up one level so it can be imported as `utils`?
```suggestion
from utils import parse_args, build_abcd_params_config
```

Reply:
Makes sense, removing the folder.